code/modules/vehicles/mecha/combat/greyscale/greyscale_constructor.dm

```diff
@@ -292,10 +291,9 @@ GLOBAL_LIST_INIT(greyscale_weapons_data, generate_greyscale_weapons_data())
 			if(isnull(selected_visor))
 				return FALSE
 			currently_assembling = TRUE
-			addtimer(CALLBACK(src, .proc/deploy_mech), 1 SECONDS)
+			addtimer(CALLBACK(src, .proc/deploy_mech), 60 SECONDS)
 			playsound(get_step(src, dir), 'sound/machines/elevator_move.ogg', 50, 0)
```

Comment:
This was intended as a visual delay to sync it up with the sound effects when it deployed
Maybe make it use TIMER_COOLDOWN instead so you just cant make successive ones instead of needing to wait